graylog2-web-interface/src/views/pages/ExtendedSearchPage.jsx

```diff
@@ -86,6 +87,7 @@ const ConnectedFieldList = connect(FieldList, { fieldTypes: FieldTypesStore, vie
   }));
 
 type Props = {
+  errors: Object[],
```

Comment:
I think this should be either `Array<Error>` or `Array<mixed>`. `Object` is not recommended for use, ever:

> Object is an alias to any and will be deprecated and removed in a future version of Flow.